### ISSUE TITLE
OS freeze when pressing PrintSCR wiht unloaded driver

### ISSUE THREAD

**Issue by Luca1991** [vitoplantamura/BugChecker#2]
Guest OS: Windows 11 Pro
Virtualizer: VMWare Workstation 17 Pro

Pressing Printscrn when the driver isn't loaded (or has been unloaded) freeze the guest OS.

**Comment by vitoplantamura** [comment]
yes this is a well known problem: some of the debugger logic needs to be copied into KDCOM: when you press PrintScr, an INT3 is executed in the kernel: KDCOM should detect this, skip the hardcoded breakpoint and resume execution. And it should do this only when BugChecker is not loaded.
